interface/main/tabs/templates/tabs_template.php

```diff
@@ -14,7 +14,7 @@
 
 <script type="text/html" id="tabs-controls">
     <div class="tabControls" data-bind="with: tabs">
-        <div class="tabNotchosen w-1">
```

Comment:
hi @ophthal , removing this class is potentially problematic. Can this be left and then just address the styling of it?

Reply:
This is because tabNotchosen is referenced by a visibility function.

Reply:
knockedout again 
![knockout](https://user-images.githubusercontent.com/5618372/110324123-8fb15780-7fe3-11eb-9192-58df5c65ba00.jpeg)

Reply:
You still need to deal with this
            <div class="tabSpan bgcolor2" data-bind="click: tabClicked, css: {tabNotchosen: !visible()}">

Reply:
@bradymiller Prob won't break tabs but will error. Changing to tabsNoHover should be okay. It's only used in this one instance...